express-LocalLibrary/routes/users.js

```diff
@@ -0,0 +1,9 @@
+var express = require('express');
+var router = express.Router();
+
+/* GET users listing. */
+router.get('/', function(req, res, next) {
+  res.send('respond with a resource');
+});
+
+module.exports = router;
```

Comment:
What is this for?

Reply:
@clarkngo did you recommend them to do this? Not quite sure what this is for?

Reply:
@lcho I had them/chuck checked this out https://expressjs.com/en/starter/generator.html

Reply:
@lcho did you have them do something else for the boilerplate?

Reply:
no, not really. Just the status page and small get request to retrieve the data from the text file. Next steps are the db debate and start on db boilerplate